src/MessageHandler/ActivityPub/Inbox/AddHandler.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\MessageHandler\ActivityPub\Inbox;
+
+use App\DTO\ModeratorDto;
+use App\Message\ActivityPub\Inbox\AddMessage;
+use App\Repository\MagazineRepository;
+use App\Service\ActivityPubManager;
+use App\Service\MagazineManager;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Messenger\Attribute\AsMessageHandler;
+
+#[AsMessageHandler]
+class AddHandler
+{
+    public function __construct(
+        private readonly ActivityPubManager $activityPubManager,
+        private readonly MagazineRepository $magazineRepository,
+        private readonly MagazineManager $magazineManager,
+        private readonly LoggerInterface $logger,
+    ) {
+    }
+
+    public function __invoke(AddMessage $message): void
+    {
+        $payload = $message->payload;
+        $actor = $this->activityPubManager->findUserActorOrCreateOrThrow($payload['actor']);
+        $targetMag = $this->magazineRepository->getMagazineFromModeratorsUrl($payload['target']);
+        if (!$targetMag) {
+            throw new \LogicException("could not find a magazine with moderators url like: '{$payload['target']}'");
```

Comment:
mentioned in matrix but wanted to track in the PR, curious if these are bubbled up to users and need translations or should be using translation tokens, or if they just go to logs

Reply:
This is in a Handler, so it will just log to a file